src/ui/mime-type-helper.js

```diff
@@ -222,7 +222,7 @@ function getCategory(pathOrExtension) {
       return CATEGORY_AUDIO;
     case 'image':
       return CATEGORY_IMAGE;
-    case 'vido':
+    case 'video':
```

Comment:
:scream:

Reply:
Seems we always showed "Unbekannt" for videos 😅 Well, now we **know**